frontend/web/styles/3rdParty/_hljs.scss

```
@@ -81,6 +82,24 @@
         color: var(--color-text-action);
       }
     }
+    // Slides and fades in when JSON is selected, collapses out otherwise.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)